server/config/config.go

```diff
@@ -33,29 +31,14 @@ type StoredConfig struct {
 	// manifest_<sha1(Manifest)>. Implementation in `store.Manifest`.
 	LocalManifests map[string]string `json:"local_manifests,omitempty"`
 
-	DeveloperMode bool `json:"developer_mode"`
-	AllowHTTPApps bool `json:"allow_http_apps"`
+	DeveloperModeOverride *bool `json:"developer_mode"`
+	AllowHTTPAppsOverride *bool `json:"allow_http_apps"`
```

Comment:
Due to the change to a pointer, the logging in https://github.com/mattermost/mattermost-plugin-apps/blob/461e9e66bb08cee1effcff459249072ce0c816ff/server/config/service.go#L272 is broken: `Storing configuration, 3 installed , 0 listed apps, developer mode 0xc000e41488, allow http apps 0xc000e41489`

Reply:
Fixed in https://github.com/mattermost/mattermost-plugin-apps/pull/440/